Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Driver.hs

```diff
@@ -185,11 +185,20 @@ data DriverAadhaarInfoRes = DriverAadhaarInfoRes
   { driverName :: Text,
     driverGender :: Text,
     driverDob :: Text,
-    driverImage :: Text
+    driverImage :: Maybe Text
   }
   deriving stock (Eq, Show, Generic)
   deriving anyclass (ToJSON, FromJSON, ToSchema)
 
+-- driver aadhaar Info api by mobile Number ----------------------------------------
+
+type DriverAadhaarInfoByPhoneAPI =
+  Capture "mobileNo" Text
+    :> "aadhaarInfobyMobileNumber"
+    :> Get '[JSON] DriverAadhaarInfoRes
```

Comment:
Just create new variable named `type DriverAadhaarInfoByPhoneReq = DriverAadhaarInfoRes ` and use that.